app/controllers/api/v1/advisories_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::V1::AdvisoriesController < Api::BaseController
+  skip_before_action :verify_authenticity_token, only: [:create]
+
+  before_action :authenticate_with_api_key, only: [:create]
+  before_action :verify_authenticated_user, only: [:create]
+  before_action :find_rubygem_by_name,      only: [:create]
+  before_action :validate_rubygem, only: [:create]
+  before_action :find_version, only: [:create]
+
+  def create
+    @advisory = @version.advisories.new(description: params[:description], title: params[:title], url: params[:url], cve: params[:cve])
```

Comment:
I am not sure. I was going through the API code and strong params are not used anywhere. I still think we should use them. After your discussion on slack, do you think we should be using strong params? or should I create a separate issue on GitHub to discuss this so that if we end up using strong params we'll update the whole API code accordingly.

Reply:
I'd suggest open a separate issue and have more maintainers chime in. It'll be a good learning for you and us.

If they don't answer, then I think what you have above is ok. I think the dangerous mass assignment one is `Model.new(params[:model])`.